docs/lifecycle/lifecycle.rst

```diff
@@ -103,6 +112,9 @@ must minimally specify:
 * What is the top level technical decision making body for the project,
   analogous to a TSC, to which the TAC should look for interfaces.
 
+Additionally, the review will confirm that the incoming project scope is within
```

Comment:
Do we have a defined scope to apply this to?  Generally, I am highly highly supportive of this requirement... but it does require us having a well defined scope.

Reply:
I think this a good requirement but the actual scope is a work item for TAC and Board - the strategic direction topic

Reply:
@bdfreeman1421 I agree that defining scope is the board's purview.  I was curious if it has defined one.  If not, we should recommend it do so.

Reply:
Do we need to change the wording here or just make sure that TAC & Board work on a scope definition for LFN?